pkg/datapath/linux/config/config.go

```diff
@@ -349,6 +349,9 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 			cDefinesMap["ENABLE_FIB_LOOKUP_BYPASS"] = "1"
 		}
 
+		if option.Config.EnableEgressRevnatAtHostOnly {
+			cDefinesMap["ENABLE_EGRESS_REVNAT_AT_HOST"] = "1"
```

Comment:
nit: I would just name it ENABLE_REVNAT_AT_HOST

Reply:
sure updated.